clap_complete/tests/testsuite/bash.rs

```diff
@@ -72,6 +72,19 @@ fn sub_subcommands() {
     );
 }
 
+#[test]
+fn custom_bin_name() {
```

Comment:
This test on this commit isn't passing.

Part of the reason for this request is so that the test shows the old behavior and the commit with the fix shows how the behavior changed.

Reply:
I must have misunderstood what you were asking for. I put the test before the fix so that the test would first catch the error and fail, and then pass once the fix gets implemented. That way you also verify the test itself.

I'm guessing that this is what you actually wanted: The test should assert the old, incorrect, behavior in the first commit and pass. In the second commit with the fix it should assert the correct behavior and pass again?